src/components/DropdownContent/index.tsx

```diff
@@ -1,4 +1,6 @@
-import React, { FC } from "react";
+import React, { FC, useCallback, useState } from "react";
+import CheckIcon from "../../assets/CheckIcon";
```

Comment:
I think, it will be better than the hardcoded SVG component.
```suggestion
import {ReactComponent as CopyIcon} from './copy-icon.svg';
```